frontend/src/metabase/admin/permissions/components/PermissionsTable/PermissionsTable.jsx

```diff
@@ -122,7 +122,9 @@ export function PermissionsTable({
 
                 {entity.permissions.map(permission => {
                   return (
-                    <PermissionsTableCell key={permission.type}>
+                    <PermissionsTableCell
+                      key={permission.type ?? permission.value}
```

Comment:
`permission.value` does not identify uniquely but I believe `type` should be always present

Reply:
It looks like `type` is actually `undefined` for collection permissions (since there's only one type I guess), but you're right, value isn't unique either. I guess we'll have to use the index.

![Screen Shot 2023-10-04 at 4 19 28 PM](https://github.com/metabase/metabase/assets/30528226/9e22a3f6-1903-4950-be41-c43fecc4ffce)